app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt

```diff
@@ -697,6 +700,14 @@ class ChatRoomFragment : Fragment(), ChatRoomView, EmojiKeyboardListener, EmojiR
         }
     }
 
+    private fun sendTypingStatus(charSequence: CharSequence) {
+        if (charSequence.isNotEmpty()) {
+            presenter.sendTyping()
```

Comment:
What happens if I remove a char?

Reply:
You need to take a look in how this is working. If you remove a `char` from the `charSequence` and the `charSequence`  is not empty (see `charSequence.isNotEmpty()`) it means that there are char in the composer otherwise there is no char in the composer, so called `presenter.sendNotTyping`.